substrate/primitives/core/Cargo.toml

```diff
@@ -52,9 +52,9 @@ blake2 = { version = "0.10.4", default-features = false, optional = true }
 libsecp256k1 = { version = "0.7", default-features = false, features = ["static-context"], optional = true }
 schnorrkel = { version = "0.11.4", features = ["preaudit_deprecated"], default-features = false }
 merlin = { version = "3.0", default-features = false }
-secp256k1 = { version = "0.28.0", default-features = false, features = ["alloc", "recovery"], optional = true }
 sp-crypto-hashing = { path = "../crypto/hashing", default-features = false, optional = true }
 sp-runtime-interface = { path = "../runtime-interface", default-features = false }
+k256 = { version = "0.13.3", features = ["alloc", "ecdsa"], default-features = false }
```

Comment:
default features are:
[`["arithmetic", "ecdsa", "pkcs8", "precomputed-tables", "schnorr", "std"]`](https://github.com/RustCrypto/elliptic-curves/blob/cdca0123ab36efaeaab92ef728a0912edcc6f164/k256/Cargo.toml#L44)

I think we shall be fine with just 'alloc' and 'ecdsa'.

Reply:
precomputed-tables should impact performance significantly if this gets used in the host.  also std of course.